src/pages/en/guides/markdown-content.md

```diff
@@ -79,6 +79,36 @@ const { frontmatter } = Astro.props;
 </html>
 ```
 
+You can also add type safety using [the `Props` type](/en/guides/typescript/#component-props) with the `MarkdownLayoutProps` helper:
```

Comment:
This is a great addition Ben, thank you!

Nit, but I wonder if “safety” is the best word here as we don’t actually do anything to check the layout gets passed the props it expects.

```suggestion
You can set a layout’s [`Props` type](/en/guides/typescript/#component-props) with the `MarkdownLayoutProps` helper:
```